ui-modules/utils/logbook/logbook.js

```diff
@@ -0,0 +1,188 @@
+
+        let vm = this;
+        let scrollableElements = Array.from($element.find('textarea'));
+
+        $scope.$on('logbook.query', () => {
+            vm.doQuery();
+        });
+
+        vm.getChecked = function (group) {
+            let levels = [];
+            group.forEach(function (item) {
+                if (item.selected) levels.push(item.value);
+            });
+            return levels;
+        }
+        vm.doQuery = function () {
+            $scope.waitingResponse = true;
+            $scope.results = "Loading..."
+
+            const levels = $scope.allLevels ? ['ALL'] : vm.getChecked($scope.logLevels);
+
+            const params = {
+                reverseOrder: $scope.reverseOrder,
+                numberOfItems: $scope.numberOfItems,
+                levels: levels,
+                initTime: $scope.initTime,
+                finalTime: $scope.finalTime,
+            }
+
+            logbookApi.logbookQuery(params, true).then(function (success) {
+                // TODO implement logic for make output as table
+                $scope.logEntries = success;
+                $scope.results = vm.createLogOutputAsText($scope.logEntries);
+                scrollToMostRecentRecords();
+            }, function (error) {
+                $scope.results = "Error getting the logs: \n" + error.error.message;
+                console.log(JSON.stringify(error));
+            }).finally(() => {
+                $scope.waitingResponse = false;
+            });
+        };
+
+        vm.createLogOutputAsText = function (success) {
+            let output = [];
+            const fieldsToShow = vm.getChecked($scope.logFields);
+            success.forEach(entry => {
+                let outputLine = [];
+                if (fieldsToShow.includes("datetime") && entry.timestamp)
+                    outputLine.push(entry.timestamp);
+                if (fieldsToShow.includes("taskId") && entry.taskId)
+                    outputLine.push(entry.taskId);
+                if (fieldsToShow.includes("entityIds") && entry.entityIds)
+                    outputLine.push(entry.entityIds);
+                if (fieldsToShow.includes("level") && entry.level)
+                    outputLine.push(entry.level);
+                if (fieldsToShow.includes("bundleId") && entry.bundleId)
+                    outputLine.push(entry.bundleId);
+                if (fieldsToShow.includes("class") && entry.class)
+                    outputLine.push(entry.class);
+                if (fieldsToShow.includes("threadName") && entry.threadName)
+                    outputLine.push(entry.threadName);
+                if (fieldsToShow.includes("message") && entry.message)
+                    outputLine.push(entry.message);
+
+                output.push(outputLine.join(" "));
+            })
+            return output.length > 0 ? output.join("\n") : "No results";
+        }
+
+        vm.resetForm = function () {
+            $scope.numberOfItems = 1000;
```

Comment:
Default number of items suggested in the query is 1000 now, instead of 10 previously.